src/piv.rs

```diff
@@ -339,8 +350,70 @@ impl RetiredSlotId {
     }
 }
 
+/// Management slot IDs.
+#[allow(missing_docs)]
+#[derive(Clone, Copy, Debug, PartialEq)]
+pub enum ManagementSlotId {
+    PIN,
+    PUK,
```

Comment:
Per the [Rust API guidelines naming conventions](https://rust-lang.github.io/api-guidelines/naming.html), these should be `Pin` and `Puk`.

Reply:
TIL: `acronyms count as one word: use Uuid rather than UUID`